odoo/fields.py

```diff
@@ -785,6 +816,9 @@ def convert_to_record(self, value, record):
         """
         return False if value is None else value
 
+    def convert_to_record_multi(self, values, record):
+        return [self.convert_to_record(value, record) for value in values]
```

Comment:
Performance: avoid the method lookup in the loop
```suggestion
        convert = self.convert_to_record
        return [convert(value, record) for value in values]
```